pom.xml

```diff
@@ -216,116 +206,18 @@
                     </configuration>
                 </plugin>
 
-                <!-- QUALITY ASSURANCE PLUGINS -->
-
-                <plugin>
-                    <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
Er vi sikre på at det ikke er noen som bruker sonar etc?

Reply:
Har gjort en liten analyse på hvem som bruker common, og det er for det meste FO og DigiSYFO som bruker common: https://confluence.adeo.no/display/INI754/Analyse+av+fellesbiblioteker

Reply:
@nutgaard Reverterte endringene som går på plugins, så kan vi heller ta det i en separat PR om vi føler behovet